src/auth.ts

```diff
@@ -132,24 +132,21 @@ export class Role {
  * be added to Roles to manage permissions.
  */
 export class User {
-  constructor(
-    private client: AuthClient,
-    public readonly name: string,
-  ) {}
+  constructor(private client: AuthClient, public readonly name: string) {}
 
   /**
    * Creates the user, with the provided password.
    */
   public create(password: string): Promise<this> {
-    return this.client.userAdd({ name: this.name, password })
-      .then(() => this);
+    return this.client.userAdd({ name: this.name, password }).then(() => this);
```

Comment:
It's for cases like this:

```ts
return client.user('connor')
  .create('password123')
  .then(user => user.addRole('root'));
```

rather than requiring

```ts
const user = client.user('connor');
return user.create()
  .then(() => user.addRole());
```

Mostly style preference, but as there isn't anything useful I'd otherwise want to return from `user.create()`, I may as well return `this` 😄